core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -849,13 +853,17 @@ public void appendTimeoutInsight(InsightBuilder insight) {
    * <p>This is, so that the InProcess transport behaves in the same way as the other transports,
    * when exchanging statuses between client and server and vice versa.
    */
-  private static Status stripCause(Status status) {
+  private static Status stripCause(Status status, boolean includeCauseWithStatus) {
```

Comment:
The method name, the argument, and the javadoc does not match.

Reply:
Not quite sure what you mean. Where is the discrepancy?

Reply:
I think he's referring to where it says "but stripped of any other information (i.e. cause)"

Reply:
> Where is the discrepancy?

The method name says "strip cause"
The javadoc says "strip something, i.e. cause"
The new argument says "do not strip cause if true".

Reply:
Ok gotcha. I'll update the javadoc